### PR TITLE
Export serde methods

### PR DESCRIPTION
**Pull request by micbakos-rdx**

### CODE REVIEW

**Review by CyonAlexRDX — APPROVED** [review]
Nice! Looks great! Maybe add some unit test for unhappy path.

Maybe you can write "Profile" as type name, no need to read the name of the type since we know it is Profile?

**Comment by micbakos-rdx** [comment]
@CyonAlexRDX Added some extensions too

File: src/profile/v100/profile.rs [hunk 1]
```diff
@@ -390,6 +415,24 @@ mod tests {
...
 
+    #[test]
+    fn to_json_bytes_new_from_json_bytes() {
+        let sut = SUT::sample();
+
+        let encoded = sut.to_json_bytes().unwrap();
+        let profile_result = SUT::new_from_json_bytes(encoded).unwrap();
+        assert_eq!(profile_result, sut);
+
+        let malformed_profile_snapshot = BagOfBytes::from("{}".as_bytes());
```
Comment (Sajjon) [inline src/profile/v100/profile.rs:426]: can you split the happy path and unhappy path unit tests in two independent tests?

File: src/wrapped_radix_engine_toolkit/low_level/transaction_classes/detailed_manifest_class.rs [hunk 2]
```diff
@@ -84,7 +84,7 @@ pub enum DetailedManifestClass {
         pool_addresses: Vec<PoolAddress>,
 
         /// The redemptions observed in the transaction
-        pool_contributions: Vec<TrackedPoolRedemption>,
+        pool_redemptions: Vec<TrackedPoolRedemption>,
```
Comment (Sajjon) [inline src/wrapped_radix_engine_toolkit/low_level/transaction_classes/detailed_manifest_class.rs:87]: thx for fixing my copy paste error!

**Pull request merged**